src/hugo/content/en-us/implementations/aws/command/command2/_index.md

```diff
@@ -0,0 +1,721 @@
+device -> broker : on initial connect/reconnect, publish("$aws/things/device1/shadow/get")
+broker -> device : send persisted shadow state, publish("$aws/things/device1/shadow/get/accepted") 
+device -> device : do-something as per command
+device -> broker : On completion of command update reported state, publish("$aws/things/device1/shadow/update", "reported : {"command": [{ "do-something":{"payload":"goes here"} }]}")
+broker -> shadow : updates reported state on shadow file
+broker -> app : publish("$aws/things/device1/shadow/update/accepted", "success")
+app -> app : reconcile("device1")
+rnote over device #FFFFFF: If device remained connected
+app -> broker : publish("$aws/things/device1/shadow/update", "desired : {"command": [{ "do-something":{"payload":"updated value"} }]}")
+broker -> shadow : updates shadow for desired command message from app
+broker -> device : publish("$aws/things/device1/shadow/update/delta", "desired : {"command": [{ "do-something":{"payload":"updated value"} }]}"")
+device -> device : do-something as per command
+device -> broker : On completion of command update reported state, publish("$aws/things/device1/shadow/update", "reported : {"command": [{ "do-something":{"payload":"updated value"} }]}")
+broker -> shadow : updates reported state on shadow file
+broker -> app : publish("$aws/things/device1/shadow/update/accepted", "success")
+app -> app : reconcile("device1")
+
+@enduml
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+### Assumptions
+
+- Used classic shadow instead of named shadow.
+- The _Device_ shadow is pre-created before the device performs the get request on connect/re-connect.
+- The _Device_ remained connected after initial connect, device waits for updates to the desired configuration or command.
+- All participants successfully receive, forward, and act up the message flows in either direction.
+
+
+## Implementation
+
+Both the _Client Application_ and the _Device_ can use similar approach while connecting, sending, and receiving messages. The code samples below are complete for each participant.
+
+{{% notice note %}}
+The code samples focus on the _command_ design in general. Please refer to the [Getting started with AWS IoT Core](https://docs.aws.amazon.com/iot/latest/developerguide/iot-gs.html) for details on creating things, certificates, and obtaining your endpoint. The code samples below are used to demonstrate the basic capability of the _Command_ pattern.
+{{% /notice %}}
+
+### Device
+
+The _Device_ code focuses on connecting to the _Broker_ and then subscribing to reserved shadow topics to receive commands and configuration changes. Upon receipt of a command, the _Device_ performs some local action and then publishes the outcome (success, failure) of the command back to the _Client Application_, which then reconcile the command. The _Device_ will continue to receive and respond to configuration updates or commands or both until stopped.
+
+{{< tabs groupId="device-code">}}
```

Comment:
the device1 tab not loaded, check if space before `>}}` corrects. Or, maybe remove tabs here completely since the application code below doesn't use one.